utils/zerovec/src/zerovec/slice.rs

```diff
@@ -251,10 +251,10 @@ where
     ///     }
     /// };
     ///
-    /// let zs_char: &ZeroSlice<char> = zs_u32.try_as_converted().expect("valid code points");
+    /// let zs_i32: &ZeroSlice<i32> = zs_u32.try_as_converted().expect("valid code points");
```

Comment:
Nit: Update the expect message (or go back to chars)

Reply:
Updated to char, [u8; 3]